src/entity-configuration/definitions/view-defs/experiment.ts

```diff
@@ -93,4 +77,56 @@ export const ViewsDefinition: { [key: string]: ViewDefinitionConfig } = {
       EntityCoreFields.CreationDate,
     ],
   },
+  [DataType.PairedNeuronCircuitSimulation]: {
+    title: 'Paired Neurons Simulation',
+    group: DataTypeGroup.SimulationData,
+    name: EntitySlug.PairedNeuronCircuitSimulation,
+    curated: false,
+    columns: [
+      EntityCoreFields.Name,
+      EntityCoreFields.Description,
+      EntityCoreFields.CircuitName,
+      EntityCoreFields.SimulationCampaignStatus,
+      EntityCoreFields.CreatedBy,
+      EntityCoreFields.CreationDate,
+    ],
+    filterableFields: [
+      EntityCoreFields.Name,
+      EntityCoreFields.Contributions,
+      EntityCoreFields.CreatedBy,
+      EntityCoreFields.CreationDate,
+    ],
+    displayableFields: [
+      EntityCoreFields.Name,
+      EntityCoreFields.Contributions,
+      EntityCoreFields.CreatedBy,
+      EntityCoreFields.CreationDate,
+    ],
+  },
+  [DataType.SmallMicrocircuitSimulation]: {
+    title: 'Small microcircuit Simulation',
```

Comment:
Should we put a capital for _microcircuit_ as you did for _Paired Neurons Simulation_?